packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -49,16 +51,23 @@ class CameraComponent extends Component {
     Viewfinder? viewfinder,
     Component? backdrop,
     List<Component>? hudComponents,
+    PostProcess? postProcess,
```

Comment:
Maybe we should take a list of children here, like for the other components? And register a query on `PostProcessComponent` so that they can used in `renderTree`? Or is it not possible to have multiple `PostProcessComponent`s?